Formula/pueue.rb

```diff
@@ -0,0 +1,66 @@
+class Pueue < Formula
+  desc "Command-line tool for managing long-running shell commands"
+  homepage "https://github.com/Nukesor/pueue"
+  url "https://github.com/Nukesor/pueue/archive/v0.1.5.tar.gz"
+  sha256 "b4d4ced6caab23193c566c34488cc9adf2da0e92777616af579f59a5e52bd7a4"
+  head "https://github.com/Nukesor/pueue.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+
+    prefix.install_metafiles
+
+    bash_completion.install "utils/completions/pueue.bash" => "pueue.bash"
+    fish_completion.install "utils/completions/pueue.fish" => "pueue.fish"
+  end
```

Comment:
```suggestion
    zsh_completion.install "utils/completions/_pueue" => "_pueue"
  end
```

Reply:
Not yet unfortunately, please see https://github.com/Nukesor/pueue/issues/57

Reply:
Ah, I see.  :(